pyiceberg/table/__init__.py

```diff
@@ -4365,6 +4365,10 @@ def _readable_metrics_struct(bound_type: PrimitiveType) -> pa.StructType:
         for manifest_list in snapshot.manifests(io):
             for manifest_entry in manifest_list.fetch_manifest_entry(io):
                 data_file = manifest_entry.data_file
+                if file_content_type == "data" and data_file.content != DataFileContent.DATA:
+                    continue
+                if file_content_type == "delete" and data_file.content == DataFileContent.DATA:
```

Comment:
What do you think about moving the file_content_type options to a constants module?

Reply:
Updated signature of _files() to use `DataFileContent` Enum, had to use a list, as there are two types in case of delete files.